src/modules/fundingManager/bondingCurveFundingManager/marketMaker/IVirtualTokenSupply.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+
+pragma solidity 0.8.19;
+
+interface IVirtualTokenSupply {
+    //--------------------------------------------------------------------------
+    // Errors
+
+    /// @notice Subtracting would result in an underflow.
+    error VirtualTokenSupply__SubtractResultsInUnderflow();
+
+    /// @notice Adding would result in and overflow.
+    error VirtualTokenSupply_AddResultsInOverflow();
+
+    //--------------------------------------------------------------------------
+    // Events
+
+    /// @notice Event emitted when virtual token supply has been set
+    event VirtualTokenSupplySet(uint indexed newSupply, uint indexed oldSupply);
```

Comment:
We currently don't know what information the UI will need. For that reason I went with indexing where possible rather then doing it to little. I would say it can stay like that. Any input? @FHieser @marvinkruse?

Reply:
Tagging @0xNuggan here as well for possible input

Reply:
I agree it can stay like that until we have clearer indications from the UI team. It's a small change after all